<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GearBoxClass</class>
 <widget class="QMainWindow" name="GearBoxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GearBox</string>
  </property>
  <property name="windowIcon">
   <iconset theme="dark" resource="../GearBox.qrc">
    <normaloff>:/GearBox/Branding/GEAR_logo_dark.ico</normaloff>:/GearBox/Branding/GEAR_logo_dark.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 63, 63);
color: rgb(255, 255, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="dockWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1431</width>
      <height>881</height>
     </rect>
    </property>
    <property name="windowTitle">
     <string>Main View</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <widget class="QWidget" name="mainView" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1431</width>
        <height>851</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/GearBox/Branding/GEAR_logo_dark.png);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionJapanese"/>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="sceneHierarchyDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>650</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Scene Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="GearBoxSceneHierarchy" name="sceneHierarchyDockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="propertiesDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>880</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="propertiesDockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="assetsDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>175</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Assets</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="assetsDockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="scenePlayerDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>160</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Scene Player</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="GearBoxScenePlayer" name="scenePlayerDockWidgetContents"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="text">
    <string>Japanese</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GearBoxSceneHierarchy</class>
   <extends>QWidget</extends>
   <header location="global">GearBoxSceneHierarchy.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GearBoxScenePlayer</class>
   <extends>QWidget</extends>
   <header location="global">GearBoxScenePlayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../GearBox.qrc"/>
 </resources>
 <connections/>
</ui>
